backend/reservations/auth.py

```diff
@@ -13,29 +13,29 @@
 def authenticate(request):
     data = request.data
     try:
-        jwt_data=data["jwt"]
+        jwt_data=data["guest"]["jwt"]
     except KeyError:
-        logger.warning("JWT information missing ip:%s", request.META['REMOTE_ADDR'])
+        logger.warning("[-] JWT information missing ip:%s", request.META['REMOTE_ADDR'])
```

Comment:
What is the meaning behind the `[-]` / `[+]` prefixes?

Reply:
easy for me to spot in logs :rofl: